h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/exec/XGBoostExecReq.java

```diff
@@ -0,0 +1,25 @@
+package hex.tree.xgboost.exec;
+
+import java.io.Serializable;
+import java.util.Map;
+
+public class XGBoostExecReq implements Serializable {
```

Comment:
Why `Serializable` and not `Freezable` ?

Reply:
working with Iced classes proved too much of a hassle for serializing simple POJOs

Reply:
also when exchaning object between cluster Iced does not work (classes can have different IDs in different clusters)

Reply:
Resolved then.